api.md

```diff
@@ -1822,6 +1823,33 @@ ORDER BY day;
 
 ## Utilities/Statistics [](utilities)
 
+### timescaledb_information.data_node [](#timescaledb_information-datanode)
+
+Get information on data nodes.
+
+|Name|Description|
+|---|---|
+| `node_name` | Data node name. |
+| `owner` | oid |
```

Comment:
Maybe be more specific about what this is an Oid for.

Reply:
Haha, I have no idea what it is for! This was a cop out - will ask on call alter to see how to address 😊

Reply:
Ok, figured this out!